Library/Homebrew/livecheck/strategy/git.rb

```diff
@@ -103,7 +103,8 @@ def self.versions_from_tags(tags, regex = nil, &block)
           tags.map do |tag|
             if regex
               # Use the first capture group (the version)
-              tag.scan(regex).first&.first
+              # This code is not typesafe unless the regex includes a capture group
+              T.unsafe(tag.scan(regex).first)&.first
```

Comment:
This is a type error that appears when deleting `String#first` from the active support rbi below (but I don't think this code uses that path, looking at call sites).